src/sentry/integrations/msteams/webhook.py

```diff
@@ -147,79 +148,6 @@ def verify_signature(request) -> bool:
 class MsTeamsWebhookMixin:
     provider: ClassVar[str]
```

Comment:
awesome!  last little bit is to nuke this dead class I think (doesn't look like it's used in getsentry fortunately!)

Reply:
That was just an editing mistake. It's already gone.

Reply:
oic I reviewed too quickly!